templates/client.ts.hbs

```diff
@@ -1,7 +1,7 @@
 import ClientConfig, { ClientConfigInit } from "./clientConfig";
 // Must not import from ./helpers/index to avoid circular dependency via ShopperLogin
 import { isBrowser } from "./helpers/environment";
-import { doFetch } from "./helpers";
+import { doFetch } from "./helpers/fetchHelper";
```

Comment:
why do we need this changes?

Reply:
`// Must not import from ./helpers/index to avoid circular dependency via ShopperLogin`

We import directly from the file to prevent any circular dependency issues